### ISSUE TITLE
Resolve high-priority WCAG 2.0 issues

### ISSUE THREAD

**Issue by anjackson** [ukwa/ukwa-ui#245]
An accessibility review raised some issues with the site that we should resolve.

High

- [x] WCAG-009 Tooltip cannot be operated with keyboard `/` tooltip

Medium

- [x] WCAG-002 No captions provided for video (8 `/about`) - Removed video
- [x] WCAG-003 Link in video inaccessible (1 `/`) - Removed video
- [x] WCAG-004 Headings not declared (everywhere including tips dialog)
- [x] WCAG-005 Heading levels skipped (8,9,10)
- [x] WCAG-006 Form input not associated with label (`/` main search box)
- [x] WCAG-007 Visual lists not semantically declared ( [x] banner, [x] footer, [x] tips, [x] breadcrumb, [x] collections lists, [x] results lists)
- [x] WCAG-008 Errors indicated by colour alone (6)
- [x] WCAG-011 Pages with duplicate titles (`/collection/*`, search page ok?)
- [x] WCAG-012 Focus order is illogical ([x] Language selector should be before menu, and  [x] Home should not be twice. [x] Footer should be in logical order.)
- [x] WCAG-013 Users not made aware of form success (7 `nominate`,10 `contact`)
- [x] WCAG-015 Help text not associated with input control (`nominate` email address bit)
- [x] WCAG-016 Parsing errors on all pages.
- [x] WCAG-017 Navigation menu not programatically determined. (not a proper nav)
- [x] WCAG-018 Tooltop is missing role (on `/`, same element as WAG-009)
- [ ] WCAG-020 Low colour contrast (1,6,7,9,10) 'featured', grey in forms, but also red alerts.
- [ ] WCAG-021 Focus not visible on interactive elements (now red outline, but not sure what to do for high contrast)

Low

- [x] WCAG-001 Unnecessary alt tag text (`/` and `/collection`)
- [x] WCAG-010 There no skip links provided (everywhere)
- [x] WCAG-014 Name field announce as required but not visibly (6,7 ? `nominate`, `contact`) no `*` and/or asterisks not explained.
- [x] WCAG-019 Breadcrumb control not programatically determined (use a proper ol nav with breadcrumb role)
- [x] AI-001 Helpful suggestions displayed in placeholder (and this disappear) (1,6,7)

**Comment by nicolabingham** [comment]
@anjackson thanks for the commits so far. Is there a beta webarchive.org.uk that we can see these on? (no worries if not).

**Comment by anjackson** [comment]
@nicolabingham you mean like https://beta.webarchive.org.uk/ ? :-) Yes, the live beta updates an hour or two after I commit changes.

Not quite ready for @crarugal yet...

**Comment by anjackson** [comment]
I believe I've tracked down the parsing errors in the course of refactoring the code.

**Comment by anjackson** [comment]
Okay, so I've resolved most of these, and once they are rolled out, they can be verified by @crarugal.

The main problem I'm left with is the low-contrast issue. In general, I've been relying on reusing standard Bootstrap accessibility tools, but this is a [known limitation of Bootstrap 4](https://getbootstrap.com/docs/4.0/getting-started/accessibility/#color-contrast). So the framework isn't helping here, and I'm not really sure how best to proceed, as I'm not very good at choosing colours and this will be quite a significant change.

**Comment by nicolabingham** [comment]
@anjackson, thanks for this, please let us know when they have been rolled out. 
We don't have to address all the issues at this time as long as we state that we're aware of them and will remedy them in due course. Mark Whiting has been helpful with the accessibility issues so we can see what he would advise re: contrast. I also have another 30 mins time with AbilityNet so I can ask them.

**Comment by anjackson** [comment]
@nicolabingham @crarugal good to go!  I also added a place-holder page for the accessibility statement.

**Comment by nicolabingham** [comment]
Thanks @anjackson ! Thanks @crarugal !

**Comment by anjackson** [comment]
Calling this ticked done, but pushing remaining issues forward to a new ticket.
